src/tz390.java

```diff
@@ -4694,7 +4759,7 @@ private void process_option(String opt_file_name,int opt_file_line,String token)
                for(int i = 0; i < machine_option_id.length; i++)                   // #503
                   {if(machine_option_id[i].equals(opt_machine))                    // #503
                      {opt_machine_optable = machines_optable[i];                   // #503
-                      opt_optable         = optables_optable[i];                   // #503
+               //**!!       opt_optable         = optables_optable[i];             // #543 I think this one needs to go ...
```

Comment:
Is this resolved yet???

Reply:
Yes, I have removed the comment. Since the program now passes all tests, the statement was indeed superfluous and has been taken out correctly.